src/pages/yeonokkang/Login/LoginBtn.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+export default class LoginBtn extends React.Component {
+  handleLoginBtn = () => {
+    this.setState({});
+  };
+
+  render() {
+    const { isActivate } = this.props;
+
+    return (
+      <div
+        className="sign-in"
+        style={
+          isActivate
+            ? { backgroundColor: 'blue' }
+            : { backgroundColor: 'lightblue' }
+        }
```

Comment:
삼항연산자 위치에 따라서 조금 더 간결하게 쓸 수 있습니다.
```suggestion
        style={{ backgroundColor: isActivate ? 'blue' : 'lightblue' }}
```

다만 이 경우 style 속성을 통해 inline 스타일을 주기 보다는 class 셀렉터를 토글하여 쓰는 것이 좋아보입니다. 이는 inline 스타일의 명시도 때문인데요. inline 스타일보다 css class selector를 사용해 스타일링 하는 걸 권장하는 이유에 대해 검색해보시고 명시도(우선순위)와 관련하여 댓글로 설명 남겨주세요 🙏 
```suggestion
        className={isActivate ? 'active' : 'inActive'}
```

Reply:
종택님 안녕하세요 :) 우선 리뷰 감사드립니다! 종택님의 도움으로 보다 명료한 코드로 탈바꿈 되었습니다😊
말씀하신 부분에 대해서도 조금 찾아보았는데요. 제가 작성한 것처럼 HTML 파일에서 각 태그에 직접적으로 주는 방법을 inline styling이라고 부르며,
이 경우(제가 그랬던 것 처럼) 코드 작성 시에는 간편할 수 있지만, 명시도 관련하여 **inline style이 가장 최우선순위를 부여받게 되므로** CSS 스타일이 깨질 수 있음을 확인했습니다. 그로 인해 유지보수 시에도 한 번 더 일해야하는 번거로움이 생길 수 있겠군요 🤓
덧붙여서, 명시도의 가장 기본 규칙에 따르면 '_inline styling > id선택자 > class 또는 속성 선택자 > 요소_' 의 순으로 정해집니다. 그러나, 아래에서 래영님이 짚어주신 것 처럼 id 선택자 또한 'Unique한 속성'이어야 하는 조건이 있으므로, class selector를 사용하여 스타일링을 하는 것이 권장되는 것을 확인할 수 있었습니다 :) 앞으로는 class 선택자를 사용하여 스타일링을 주도록 하겠습니다😊